fuzzers/qemu_systemmode/src/fuzzer_sync_exit.rs

```diff
@@ -52,11 +53,15 @@ pub fn fuzz() {
         // Initialize QEMU
         let args: Vec<String> = env::args().collect();
         let env: Vec<(String, String)> = env::vars().collect();
+
         // let emu_snapshot_manager = QemuSnapshotBuilder::new(true);
         let emu_snapshot_manager = FastSnapshotManager::new(false); // Create a snapshot manager (normal or fast for now).
         let emu_exit_handler: StdEmulatorExitHandler<FastSnapshotManager> =
             StdEmulatorExitHandler::new(emu_snapshot_manager); // Create an exit handler: it is the entity taking the decision of what should be done when QEMU returns.
-        let emu = Emulator::new(&args, &env, emu_exit_handler).unwrap(); // Create the emulator
+
+        let cmd_manager = StdCommandManager::new();
+
+        let emu = Emulator::new(&args, &env, emu_exit_handler, cmd_manager).unwrap(); // Create the emulator
```

Comment:
Would a builder make more sense here? What's the `CommandManager`'s use case?

Reply:
yes, I am coincidentally writing the builder for `Emulator`. i'll add it to this PR when it's done.